tests/src/test/java/org/eclipse/hono/tests/client/ClientTestBase.java

```diff
@@ -63,13 +40,13 @@
      * @param senderHandler The handler to call for sending messages to the messaging network.
      * @throws InterruptedException if test execution is interrupted.
      */
-    protected void doUploadMessages(final TestContext context, final Handler<CountDownLatch> consumerHandler, final Handler<String> senderHandler) throws InterruptedException {
+    protected void doUploadMessages(final TestContext context, final Handler<CountDownLatch> consumerCreationHandler, final Handler<String> senderHandler) throws InterruptedException {
```

Comment:
`@param consumerHandler` does not match param name used in signature. The description of the param is also wrong. The desription of the params should also indicate, what the type argument is representing, e.g. I suppose that the senderHandler will handle a message's payload, right?

FMPOV it would be more intuitive to use `java.util.function.Consumer` instead of `Handler` and then call the parameters *receiver* and *sender*, don't you think?